### ISSUE TITLE
Having options of weather articles show on public timeline

### ISSUE THREAD

**Issue by cybergipsy2020** [Plume-org/Plume#811]
**Is your feature request related to a problem? Please describe.**
I don't want articles that contains nsfw content to be shown on public timelines.

**Describe the solution you'd like**
It would be great to have an option of visibility when posting articles. Some options could be: Public, unlisted, private/only- by-link, password protected. 

**Describe alternatives you've considered**
Visibility of blogs instead of articles?

**Additional context**
How does that work with Mastodon? I know that writefreely has similar features but not sure how exactly.

**Comment by mathstuf** [comment]
Indeed, it seems that all posts are currently available publicly. It looks like `PUBLIC_VISIBILITY` is the token to search for where to actually change this in the code. Note that instead, specific ActivityPub collection URLs would be needed. I suspect some UI work to make such a thing more easily selectable would be required (though I'd be fine with just using the author(s) as an easy boolean for now).
